Src/Support/IntegrationTests/OidcTests.cs

```diff
@@ -48,5 +49,31 @@ public async Task OidcTokenProvider_ServiceAccountCredential()
             Assert.NotNull(payload);
             Assert.Contains("https://this.is.a.test", payload.AudienceAsList);
         }
+
+        [OnGceFact]
```

Comment:
Is it with adding the attribute, rather than just making this a SkippableFact and calling Skip if we're not running on GCE?

Reply:
SkippableFact is not available on netcoreapp1*. I wrote this attribute myself and I think it's right.